modules/ROOT/pages/push/coding.adoc

```diff
@@ -57,17 +174,52 @@ func application(_ application: UIApplication,
 
 ****
 
+*Using plain Cordova*
+
 [source,javascript]
 ----
+const push = PushNotification.init({
```

Comment:
This snippet could and should be polished. I will take a look and send you a suggestion.